roshan-bajgain/webpack-setup
# Testing1

## Testing
- Use jest framework for testing.
- Create a test file ([..].test.js) for a file containing the add item and delete item functions that you want to test.
- Make sure you group your tests using the description() method.
